presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -89,6 +89,12 @@ public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> ta
         return createQueryRunner(tables, extraProperties, "sql-standard", ImmutableMap.of(), baseDataDir);
     }
 
+    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Map<String, String> extraHiveProperties, Optional<Path> baseDataDir)
+            throws Exception
+    {
+        return createQueryRunner(tables, extraProperties, "sql-standard", extraHiveProperties, baseDataDir);
+    }
+
```

Comment:
Why weren't the existing factories good enough?

Reply:
We need to pass extraHiveProperties.  We could also use the full createQueryRunner factory, but this is simply a design choice.  The short of why I decided to add a factory was that this seemed to be the approach others took here.

I don't think it matters too much either way, but if I didn't want to write my own factory, I should remove the other less used factories here too.  If I had more experience in this code base and stronger opinions on Java style, I'd have a stronger reason to go one way or another.

Reply:
Hmmm... I'm ambivalent about this. @bhhari? @arhimondr  ? What do you think?

Reply:
I don't have any strong preference. Another factory method is fine.